Python/readme.md

```diff
@@ -0,0 +1,48 @@
+# QiSpace Python Demos
+
+## Sequr
+
+### Setup
+  1. Place your Qeep `.whl` library file under `lib` folder. If you do not have one, please contact Quantropi.
+  1. Install demo dependencies and libraries: `./install.sh`
+
+### Running the demo
+
+  1. **Generate key demo:**
+      - Execute the `demo_sequr_key_gen.py` with the bellow aruments:
+        ```
+        usage: demo_sequr_key_gen.py [-h] [--url URL] [--token TOKEN] [--key_size KEY_SIZE]
+
+        options:
+          -h, --help           show this help message and exit
+          --url URL            URL for QiSpace Enterprise API. ex: https://enterprise.staging.qispace.info/kds/api/v1
+          --token TOKEN        Device token generated from QiSpace Enterprise
+          --key_size KEY_SIZE  Key size to generate
+        ```
+      - You might get an output as bellow:
+        ```
+        --- Generated QK with key_id 0d0ed476-6af7-49b3-a3c1-49de2ac98798
```

Comment:
Please correct and make term "QK" and "key".  Util APIs are used to generate "Key", or raw_key as util code indicated